src/components/DatePicker/DatePicker.scss

```diff
@@ -7,30 +7,32 @@ $font-size: rem(12px);
 $in-range-border-color: #9ca6de;
 
 $range-end-border-radius: rem(30px);
+$month-width: rem(240px);
 
 .DatePicker {
   position: relative;
 }
 
 .MonthContainer {
   display: flex;
-  flex-wrap: wrap;
```

Comment:
This is the main change that fixed the layout for me in the codesandbox. I am low confidence that this will work or could have side effects 😬